libxrdp/xrdp_sec.c

```diff
@@ -374,9 +374,42 @@ xrdp_load_keyboard_layout(struct xrdp_client_info *client_info)
     char keyboard_cfg_file[256] = { 0 };
     char rdp_layout[256] = { 0 };
 
-    LOG(LOG_LEVEL_INFO, "xrdp_load_keyboard_layout: keyboard_type [%d] keyboard_subtype [%d]",
-        client_info->keyboard_type, client_info->keyboard_subtype);
+    const struct xrdp_keyboard_overrides *ko =
+            &client_info->xrdp_keyboard_overrides;
 
+    LOG(LOG_LEVEL_INFO, "xrdp_load_keyboard_layout: Keyboard information sent"
+        " by the RDP client, keyboard_type:[0x%02X], keyboard_subtype:[0x%02X],"
```

Comment:
If there is a clearer and more appropriate wording, please suggest it.

Reply:
Thank you.
I will leave this text as it is.

Reply:
keylayout value is not logged here. Is this enough?

Reply:
No, it is logged. Sorry for the noise.